ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -441,16 +441,22 @@ private object RecalculateFormula(string fA1)
                 cAddress = fA1;
             }
 
-            if (Worksheet.Workbook.WorksheetsInternal.Any<XLWorksheet>(
-                w => String.Compare(w.Name, sName, true) == 0)
-                && XLHelper.IsValidA1Address(cAddress)
-                )
-            {
-                var referenceCell = Worksheet.Workbook.Worksheet(sName).Cell(cAddress);
-                if (referenceCell.IsEmpty(XLCellsUsedOptions.AllContents))
-                    return 0;
-                else
-                    return referenceCell.Value;
+            if (Worksheet.Workbook.Worksheets.Contains(sName)
```

Comment:
Please make this change (to use `.Contains` at line 470 too). Bonus points if you can put this change in a separate commit.

Reply:
Hmm, I see wee have many cases where we use `String.Compare(String, String, Boolean)` (not only for worksheet names, but also for named ranges). Maybe this warrants a new PR?

Reply:
I've extracted these changes in a separate commit and replaced all `.Any(x => String.Compare(...))` with the direct access, where applicable.